install.py

```diff
@@ -4,12 +4,12 @@
 from setup import ExtensionInstaller
 
 def loader():
-    return ExfoliationInstaller()
+    return BelchertownInstaller()
 
-class ExfoliationInstaller(ExtensionInstaller):
+class BelchertownInstaller(ExtensionInstaller):
     def __init__(self):
-        super(ExfoliationInstaller, self).__init__(
-            version="1.0.1",
+        super(BelchertownInstaller, self).__init__(
+            version="1.1b3",
```

Comment:
Before you merge shouldn't you update the version?

Reply:
Yes, and I will. The PR gets updated automatically with every commit. I haven't committed the final 1.1 yet, we're still in 1.1b3 in development.